include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDETypeStateAnalysis.h

```diff
@@ -164,32 +164,32 @@ class IDETypeStateAnalysis
    *
    * @note Only one-level lattice's are handled currently
    */
-  l_t join(l_t lhs, l_t rhs) override;
+  l_t join(l_t Lhs, l_t Rhs) override;
 
   std::shared_ptr<EdgeFunction<l_t>> allTopFunction() override;
 
-  void printNode(std::ostream &os, n_t N) const override;
+  void printNode(std::ostream &OS, n_t Stmt) const override;
 
-  void printDataFlowFact(std::ostream &os, d_t d) const override;
+  void printDataFlowFact(std::ostream &OS, d_t Fact) const override;
 
-  void printFunction(std::ostream &os, f_t m) const override;
+  void printFunction(std::ostream &OS, f_t Func) const override;
 
-  void printEdgeFact(std::ostream &os, l_t l) const override;
+  void printEdgeFact(std::ostream &OS, l_t L) const override;
 
   void emitTextReport(const SolverResults<n_t, d_t, l_t> &SR,
                       std::ostream &OS = std::cout) override;
 
   // customize the edge function composer
   class TSEdgeFunctionComposer : public EdgeFunctionComposer<l_t> {
   private:
-    l_t botElement;
+    l_t BotElement;
 
   public:
     TSEdgeFunctionComposer(std::shared_ptr<EdgeFunction<l_t>> F,
-                           std::shared_ptr<EdgeFunction<l_t>> G, l_t bot)
-        : EdgeFunctionComposer<l_t>(F, G), botElement(bot){};
+                           std::shared_ptr<EdgeFunction<l_t>> G, l_t Bot)
```

Comment:
We probably should move `F` and `G` into the `EdgeFunctionComposer`

Reply:
True. But this would also require more refactoring due to the types of the member variables in `EdgeFunctionComposer`.